README.md

```diff
@@ -126,3 +126,53 @@ client = h2osteam.clients.DriverlessClient()
 
 ...
 ```
+
+### H2O Cloud Discovery support
+
+I you use token providers to access H2O.ai services running on H2O Cloud, you
```

Comment:
`If you use the token provider to access H2O.ai services in your H2O AI Cloud environment, you can simply the configuration by using the `h2o-authn[discovery]` library.

Reply:
Adressed.